### DISCUSSION TITLE
in RadioGroup : radio circles do not appear

### DISCUSSION THREAD

**Discussion by saqlainzaheer** [shadcn-ui/ui#4254] [Q&A]
Issue Description:
When using the RadioGroup and RadioGroupItem components from ShadCN UI with the default setup, the radio circles do not appear, and only the radio text is visible. This requires additional custom CSS to make the radio circles visible, which may indicate a missing default style configuration. is any one facing this issue ?

**Comment by chrisauret** [comment]
Make sure you are **not** importing components from `@radix-ui`
Your imports should look something like 
`import { RadioGroup, RadioGroupItem } from "@/components/ui/radio-group";`
